doc/sklearn.rst

```diff
@@ -52,6 +54,40 @@ on all added and monkey-patched classes, which are discovered by means of
 introspection. This assures scikit-learn API compatibility of all
 `daal4py.sklearn` classes.
 
+.. _sklearn_verbose:
+
+scikit-learn verbose
+----------------
+
+In order to find out which implementation of the algorithm is currently used,
```

Comment:
you can just use "to" instead of "in order to" (it's recommended by the style guide)